dynamodb/package.json

```diff
@@ -11,6 +11,10 @@
     "url": "https://github.com/winglang/winglibs.git",
     "directory": "dynamodb"
   },
+  "files": [
+    "*.w",
+    "*.mjs"
+  ],
```

Comment:
Wing should automatically be adding these as part of the CLI. *.w is already included, and *.mjs should be fixed upstream

Reply:
We need to be careful with the defaults, though. We might end up including config files and other unrelated stuff (ie, eslint config)

Reply:
Maybe we should discuss wing pack altogether? How does it differ from npm pack?

Reply:
For now it's primarily an opinioned set of defaults/checks. Ideally we don't want to rely on the user knowing all about the various package.json setting they need to handle to publish something.